tests/tests/assets.rs

```diff
@@ -0,0 +1,117 @@
+use fuel_core::{
+    chain_config::{
+        CoinConfig,
+        StateConfig,
+    },
+    database::Database,
+    service::{
+        Config,
+        FuelService,
+    },
+};
+use fuel_core_client::client::{
+    types::{
+        primitives::AssetId,
+        TransactionStatus,
+    },
+    FuelClient,
+};
+use fuel_core_types::fuel_tx::{
+    Input,
+    Output,
+    TransactionBuilder,
+    TxId,
+};
+
+async fn setup_service(configs: Vec<CoinConfig>) -> FuelService {
+    let state = StateConfig {
+        coins: configs,
+        ..Default::default()
+    };
+    let config = Config::local_node_with_state_config(state);
+
+    FuelService::from_database(Database::default(), config)
+        .await
+        .unwrap()
+}
+
+#[tokio::test]
+#[ignore] // TODO: Need to be able to mint assets for this test to work
```

Comment:
I'm not quite sure how to enable permissions to be able to mint assets for the purposes of this test